tests/compute/test_frame.py

```diff
@@ -181,7 +183,7 @@ def test_row2():
         rowid = Index(F.tensor([0, 2]))
         rows = f[rowid]
         y = rows['a1']
-    F.backward(y, F.ones((len(rowid), D)))
+        F.backward(y, F.ones((len(rowid), D)))
```

Comment:
Just a question. Is there any difference when putting F.backward in the scope?

Reply:
In tensorflow, the backward is like the code below:
```
with tf.GradientTape() as tape:
      grads = tape.gradients(loss, model.trainable weights)
```
You can see the corresponding `backward` function is `tape.gradients` which needs to be in the scope under my implementation.